jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -519,3 +524,13 @@ def _live_update(self, event={}):
         for mark in self.marks.values():
             mark.update_xy(sp.spectral_axis.value, sp.flux.value)
             mark.visible = True
+
+    def translate_units(self, collapsed_spec):
```

Comment:
this is probably fine for this PR to demonstrate the translation, but when we connect a switch, we'll want to have the ability to change the entry already loaded in the data-collection (preferably in-place without having to re-parse).